microservices/frontend/src/modules/admin/AdminPage.js

```diff
@@ -1,4 +1,59 @@
+import { useEffect, useState } from 'react';
+import api from '../../api';
+import { Container} from 'reactstrap';
+import { ListUser } from './components/ListUser';
+import { toast } from 'react-toastify';
+
 
 export function AdminPage () {
-  return <h1>Admin page</h1>;
+  const [state, setState] = useState({
+    loaded: false,
+    users: []
+  });
+
+  useEffect(() => {
+    loadUsers({ state, setState });
+  }, []);
+
+  async function onUserDelete ({ user }) {
+    setState((state) => ({ ...state, loaded: false }));
+    try {
+        await api.identity.delete(user.id);
+        setState((state) => ({users: state.users.filter((u) => u.id !== user.id), loaded: true }));
+    } catch (e) {
+      setState((state) => ({ ...state, loaded: true }));
+      toast.error(`Error al borrar el usario. ${e.message}`);
+    }
+  }
+
+  async function onUserEdit (newUser, user) {
+    console.log(newUser);
+    console.log(user);
+    //setState((state) => ({ ...state, loaded: false }));
```

Comment:
remove death code. i.e comments you're not using.

Reply:
also, remove consoles logs

Reply:
Done :)